### PR TITLE
Pool rewards tests

### PR DESCRIPTION
**Pull request by AlexChetverov**

### CODE REVIEW

File: test/parallel/xyk-pallet.API.pool.weightedPools.test.ts [hunk 1]
```diff
@@ -179,6 +181,107 @@ test("Testing that the sum of the weights can be greater than 100", async () =>
...
+
+  const [testUser2] = setupUsers();
+
+  await Sudo.batchAsSudoFinalized(
+    Assets.mintToken(token1, testUser2, Assets.DEFAULT_AMOUNT),
+    Assets.mintNative(testUser2)
+  );
+
+  const totalIssuanceBefore = new BN(
+    await api.query.tokens.totalIssuance(liqId)
```
Comment (goncer) [inline test/parallel/xyk-pallet.API.pool.weightedPools.test.ts:244]: have you tried usign this? 
issuance.promotedPoolsRewardsV2()

are not the rewards growing on each pool?  IDK what is totalIssuance about :(

**Review by goncer — CHANGES_REQUESTED** [review]

File: test/parallel/xyk-pallet.API.pool.weightedPools.test.ts (hunk 1 above)
Comment (AlexChetverov) [reply test/parallel/xyk-pallet.API.pool.weightedPools.test.ts]: The function was replaced, I also changed the async function which receives information about the pool's rewards and weight

File: test/parallel/xyk-pallet.API.pool.weightedPools.test.ts [hunk 2]
```diff
@@ -174,19 +179,114 @@ test("Testing that the sum of the weights can be greater than 100", async () =>
...
+    token1,
+    defaultCurrencyValue
+  );
+
+  await testUser2.refreshAmounts(AssetWallet.AFTER);
+
+  expect(testUser2.getAsset(MGA_ASSET_ID)!.amountAfter.free!).bnGt(BN_ZERO);
+  expect(testUser2.getAsset(MGA_ASSET_ID)!.amountAfter.reserved!).bnEqual(
+    BN_ZERO
+  );
```
Comment (goncer) [inline test/parallel/xyk-pallet.API.pool.weightedPools.test.ts:231]: //TODO: Goncer, check if there is sth about issuance.

Comment (goncer) [reply test/parallel/xyk-pallet.API.pool.weightedPools.test.ts:231]: @goncer

**Review by goncer — CHANGES_REQUESTED** [review]
need to doubleCheck on thing.

**Review by goncer — APPROVED** [review]

**Pull request merged**